tests/test_reporters.py

```diff
@@ -705,3 +705,43 @@ def test_github_annotation_report(
         )
         == expected_custom_output
     )
+
+
+@pytest.mark.parametrize(
+    "report1, report2, expected_default_output, expected_custom_output",
```

Comment:
Can we write 2 parametrized tests, one for the default config and one for the custom config? Something like:

```python
"report1, report2, config, expected_default_output, expected_custom_output"
```

... and then pass the config dict as parametrized input to the test.

Reply:
[@fce](https://github.com/aconrad/pycobertura/pull/166/commits/feb16f4e45ffb33f0ff179e575846b49adca4e26)